src/Bouncefix.js

```diff
@@ -1,4 +1,6 @@
 var React = require('react');
+var ReactDOM = require('react-dom');
+var assign = require('object-assign');
```

Comment:
probably a good idea. React depends on 'object-assign' starting with v0.14 so I didn't notice any problems with this in my own projects.